.github/workflows/ci.yml

```diff
@@ -21,10 +18,6 @@ jobs:
         if: github.event_name == 'push'
         run: |
           gitlint
-      - name: Lint All Commits on Pull Request
```

Comment:
Removing this will mean `gitlint` only checks the last commit, not all commits on a PR. I guess if kodiak is always used / configured to squash and merge, this is okay?

Reply:
Relevant: https://github.com/linz/template-python-hello-world/issues/36

Reply:
this takes out the `pull_request` event, I'm not sure we want to have both build and `pull_request` as everything will double up if you push to this repo and make a PR.